CONTRIBUTING.md

```diff
@@ -0,0 +1,60 @@
+# Contributing
+
+## Code of Conduct
+
+Please note that this project is released with a [Code of Conduct](./CODE_OF_CONDUCT.md).
+By participating in this project you agree to abide by its terms.
+
```

Comment:
Something about "all contributors are working on this project in their free time" and that immediate reactions are not to be expected.

Reply:
Not sure if that should be in the CoC instead/also.

Reply:
Good idea! I put this in the introduction.